starlingcaptureapi/asset_helper.py

```diff
@@ -1,222 +1,176 @@
-            _logger.info("Created internal assets directory: " + dir_internal_assets)
-        if _file_util.create_dir(dir_internal_claims):
-            _logger.info("Created internal claims directory: " + dir_internal_claims)
-        if _file_util.create_dir(dir_internal_tmp):
-            _logger.info(
-                "Created internal temporary assets directory: " + dir_internal_tmp
-            )
-        if _file_util.create_dir(dir_internal_create):
-            _logger.info(
-                "Created internal assets create directory: " + dir_internal_create
-            )
-        if _file_util.create_dir(dir_internal_create_proofmode):
-            _logger.info(
-                "Created internal assets create-proofmode directory: "
-                + dir_internal_create_proofmode
-            )
-        if _file_util.create_dir(dir_add):
-            _logger.info("Created shared assets add directory: " + dir_add)
-        if _file_util.create_dir(dir_update):
-            _logger.info("Created shared assets update directory: " + dir_update)
-        if _file_util.create_dir(dir_store):
-            _logger.info("Created shared assets store directory: " + dir_store)
-        if _file_util.create_dir(dir_custom):
-            _logger.info("Created shared assets custom directory: " + dir_custom)
-        if _file_util.create_dir(dir_create_output):
-            _logger.info(
-                "Created shared assets create output directory: " + dir_create_output
-            )
-        if _file_util.create_dir(dir_create_proofmode_output):
-            _logger.info(
-                "Created shared assets create-proofmode output directory: "
-                + dir_create_proofmode_output
-            )
-        if _file_util.create_dir(dir_add_output):
-            _logger.info(
-                "Created shared assets add output directory: " + dir_add_output
-            )
-        if _file_util.create_dir(dir_update_output):
-            _logger.info(
-                "Created shared assets update output directory: " + dir_update_output
-            )
-        if _file_util.create_dir(dir_store_output):
-            _logger.info(
-                "Created shared assets store output directory: " + dir_store_output
-            )
-        if _file_util.create_dir(dir_custom_output):
-            _logger.info(
-                "Created shared assets custom output directory: " + dir_custom_output
+    def __init__(self, organization_id):
+        """
+        Args:
+            organization_id: string uniquely representing an organization
+                must not contain spaces or special characters, as it will become
+                part of directory names (e.g. "hyphacoop" good, not "Hypha Coop")
+        """
+        if self._filename_safe(organization_id) != organization_id:
+            raise ValueError(
+                f"Organization {organization_id} does not to be filename safe!"
```

Comment:
```suggestion
                f"Organization {organization_id} does not have to be filename safe!"
```
Not sure of logic if `does not have` or `does have`

Reply:
Oops, yes, I fixed the error. It was meant to say "it is not filename safe".